### ISSUE TITLE
Add list of soft deleted post in MCP (with actions)

### ISSUE THREAD

**Issue by effone** [mybb/mybb#3160]
Similar to the unapproved posts under "Moderation Queue" list down all soft deleted posts / threads with batch actions (restore, delete). May appear inside "Moderation Queue" itself or as a separate listing ...

**Comment by kat-kta** [comment]
I'd argue that a togglable Recycle Bin option where you would declare a Trash forum (where all deleted posts would be moved to instead of being deleted) would be much better. I personally moved my forum from SMF to MyBB and the SMF team really hit the spot with how they designed this interaction with the deletion of posts.

**Comment by effone** [comment]
Not at all a big deal, possibilities are immense. But that may be the nature how MyBB works. We decide to implement anything based on cimmunity discussion. What you are pointing out is kinda off-topic here. Better post your opinion in suggestions and feedback section in forums.

**Comment by euantorano** [comment]
Yep, as said above were open to suggestions, but a bigger feature change like that would be more likely to happen in a release like 1.10 alongside other new features.

**Comment by Ben-MyBB** [comment]
@arcalys ?

**Comment by Sama34** [comment]
You can already search for soft-deleted threads/posts using the search feature.
